pkg/liqoctl/install/aks/provider.go

```diff
@@ -159,6 +161,8 @@ func (o *Options) parseClusterOutput(ctx context.Context, cluster *containerserv
 	}
 
 	switch {
+	case o.privateLink:
+		o.APIServer = *cluster.PrivateFQDN
```

Comment:
Can you please check that the value is not nil and return an error in this case?